rest_framework_simplify/views.py

```diff
@@ -324,6 +340,18 @@ def get(self, request, pk=None, parent_resource=None, parent_pk=None):
             # exclude any items that shouldnt be in the final list
             obj = obj.using(self.read_db).exclude(**exclude_filter_kwargs)
 
+        # handle aggregates
+        aggregate = request.query_params.get('aggregate', None)
+ 
+        if aggregate:
+            raw_type, raw_field = aggregate.split('___')
```

Comment:
Do you want explicit error handling to give a more specific error message or do you just want to let a natural exception happen?